tests/pytorch/test_sanity.py

```diff
@@ -518,7 +518,7 @@ def test_sanity_bert(dtype, bs, fp8_recipe, model, skip_wgrad, zero_centered_gam
             apply_residual_connection_post_layernorm=True,
             output_layernorm=True,
             zero_centered_gamma=zero_centered_gamma,
-            self_attn_mask_type="padding",
+            self_attn_mask_type="causal",
```

Comment:
Why?

Reply:
Reverted. There was a reason for this when I made the change but I suspect #565 has fixed it possibly.